lib/packages/docutils/rstast.nim

```diff
@@ -77,6 +77,11 @@ type
 
   PRstNode* = ref RstNode    ## an RST node
   RstNodeSeq* = seq[PRstNode]
+  PRstLocation* = ref object ## location inside a file
+    filename*: string
```

Comment:
I would try to reuse the `TLineInfo` mechanism.

Reply:
see also `NimLineInfo` refs https://github.com/nim-lang/RFCs/issues/387